### PR TITLE
Add data set quality docs to stateful

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
Add the log data set quality docs to stateful.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-fnwi7m6dy-elastic-dev.vercel.app

* Built with commit db1e7df9e723f6433153e9258e6611aba1bc438d

> Issues? Visit #next-docs in Slack

File: docs/en/observability/logs-monitor-datasets.asciidoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+[[monitor-datasets]]
+= Monitor log data set quality
+
+beta:[]
+
+The **Data Set Quality** page provides an overview of your log data sets.
+Use this information to get an idea of your overall log data set quality and find data sets that contain incorrectly parsed documents.
+Access the Data Set Quality page from the main {kib} menu at **Stack Management** → **Data Set Quality**.
```
Comment (alaudazzi) [inline docs/en/observability/logs-monitor-datasets.asciidoc:8]:
```suggestion
Access the **Data Set Quality** page from the main {kib} menu at **Stack Management** → **Data Set Quality**.
```

File: docs/en/observability/logs-monitor-datasets.asciidoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Degraded (image::../images/yellow-dot-icon.png[Degraded icon]): Greater than 0% and up to 3% of the documents in the data set are degraded.
+* Poor (image::../images/red-dot-icon.png[Poor icon]): Greater than 3% of the documents in the data set are degraded.
+
+Opening the details of a specific data set shows the degraded documents history, a summary for the data set, and other details that can help you determine if you need to investigate any issues.
+
+[discrete]
+[[investigate-issues]]
+== Investigate issues
+
+The Data Set Quality page has a couple of different ways to help you find ignored fields and investigate issues.
```
Comment (alaudazzi) [inline docs/en/observability/logs-monitor-datasets.asciidoc:37]:
```suggestion
The **Data Set Quality** page has a couple of different ways to help you find ignored fields and investigate issues.
```

File: docs/en/serverless/logging/monitor-datasets.mdx [hunk 3]
```diff
@@ -6,12 +6,11 @@ description: Monitor log data sets to find degraded documents.
 tags: [ 'serverless', 'observability', 'how-to' ]
 ---
 
-<p><DocBadge template="technical preview" /></p>
 <p><DocBadge template="beta" /></p>
 
 The **Data Set Quality** page provides an overview of your log data sets.
 Use this information to get an idea of your overall log data set quality and find data sets that contain incorrectly parsed documents.
-Access the Data Set Quality page from the main ((kib)) menu and go to **Stack Management** → **Data Set Quality**.
+Access the Data Set Quality page from the main menu at **Project settings** → **Management** → **Data Set Quality**.
```
Comment (alaudazzi) [inline docs/en/serverless/logging/monitor-datasets.mdx:13]:
```suggestion
Access the **Data Set Quality** page from the main menu at **Project settings** → **Management** → **Data Set Quality**.
```

**Review by alaudazzi — APPROVED** [review]
A few minor suggestions, otherwise LGTM.

**Pull request merged**